broker/Dockerfile

```
@@ -1,5 +1,5 @@
 # FROM node:14.16-alpine3.13 also works here for a smaller image
-FROM node:16.10-alpine3.14
+FROM node:16.14-alpine3.14
```

```suggestion
FROM node:16.14-alpine3.15
```
I think alpine 3:15 is available.

---

Done